usdigitalresponse/usdr-gost
# Upgrading AWS SDK to v3

Added AWS SDK v3 libraries, removed legacy aws-sdk. 

I'm rusty in the stack here and appreciate any feedback!

Closes #653.

### Ticket #653
## Description
Migrates AWS SDK to v3


## Screenshots / Demo Video

## Testing
1. Launch locally with localstack using configured values added in .env.example
2. Request login link. Email should be found in volume/tmp/state/ses/.
3. Navigate to /arpa_reporter
4. Click "Send audit report by email"
5. Aforementioned email location should include an email with localstack S3 link.

### Automated and Unit Tests
- [ ] Added Unit tests **ed note, modified unit tests**

### Manual tests for Reviewer

## Checklist
- [ ] Provided screenshots/demo (**ed note, no functional change**)
